doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1,9 +1,27 @@
 FORMAT: 1A
-HOST: http://orion.lab.fiware.org/
+HOST: http://telefonicaid.github.io/fiware-orion/api/v2/
```

Comment:
fixed in https://github.com/jmcanterafonseca/fiware-orion/commit/01c2427080c7f44b53a89e26ce38de3faf33d990